test/xUnit/csharp/test_CompletionHelpers.cs

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
@ArmaanMcleod Let's use this PR only to create a new class. All other changes should be in separate PRs.

Reply:
Please add new tests in new PR. Only then refactor in another PR.

Reply:
@iSazonov Seems interesting how tests should be in another PR. I guess I am trying to make sure moved code has tests so refactoring in other PRs doesn't cause regression.

Reply:
Tests should work for current code in main branch. Then they should continue to work after change.